src/main/java/cn/nukkit/Server.java

```diff
@@ -287,6 +287,20 @@ public Level remove(Object key) {
             String fallback = BaseLang.FALLBACK_LANGUAGE;
             String language = null;
             while (language == null) {
+                if(predefinedLanguage != null)  {
+                    log.info("Trying to load language from predefined language: " + predefinedLanguage);
+
+                    InputStream conf = this.getClass().getClassLoader().getResourceAsStream("lang/" + predefinedLanguage + "/lang.ini");
+                    if (conf != null) {
+                        language = predefinedLanguage;
+                        log.info("Found language for predefined language: " + predefinedLanguage);
+                        break;
+                    } else {
+                        log.warn("No language found for predefined language: " + predefinedLanguage + ", please choose a valid language");
+                    }
+                    predefinedLanguage = null;
+                }
```

Comment:
Wouldn't it be much simpler if we didn't duplicate the code like so:
```java
String lang;
if (predefinedLanguage != null) {
    lang = predefinedLanguage;
    predefinedLanguage = null;
} else {
    lang = this.console.readLine();
}

Reply:
The reason I did it this way is because we have a little bit more messages towards the enduser now, otherwise if the language is invalid there is just an empty line with nothing (it expects you enter a language)